onebusaway-android/src/main/java/org/onebusaway/android/tripservice/NotifierTask.java

```diff
@@ -74,6 +79,8 @@ public NotifierTask(Context context,
         mUri = uri;
         mNotifyTitle = notifyTitle;
         mNotifyText = notifyText;
+
+        mSettings = Application.getPrefs();
```

Comment:
Does this need to be a member variable?   It is only used once and comes from a static method so it could just be a local variable down in createNotification().

Reply:
I can make it local